src/utils/apiEndpoint.js

```diff
@@ -12,7 +12,7 @@ const getApiEndpoint = () => {
 
     return 'https://api.scp.biomage.net';
   } catch (error) {
-    console.error('Failed to get API endpoint', window.location.href);
+    console.error('Failed to get API endpoint.');
   }
 };
```

Comment:
For server-side use we make sure we can call this API with a custom `location` parameter, because on server-side there are no `window` objects.